### ISSUE TITLE
[검수] Ch 11 — BERT Binary 방식 B — softmax + CrossEntropyLoss

### ISSUE THREAD

**Issue by yoon-gu** [yoon-gu/neuqes-101#35]
## 담당과 일정

- 기본 페어: A
- 세 번째 검수자: 해당 없음
- Phase: Phase 1
- Required reviews: 2
- Target date: 2026-09-13

## 기준 버전

- Source notebook: `11_bert_binary_softmax/11_bert_binary_softmax.ipynb`
- Source commit:
- WikiDocs URL:
- EPUB build:

## Notebook

- [ ] 1차 검수자 PASS
- [ ] 2차 검수자 PASS

## WikiDocs

- [ ] 1차 검수자 PASS
- [ ] 2차 검수자 PASS

## EPUB

- [ ] 1차 검수자 PASS
- [ ] 2차 검수자 PASS

## 완료 조건

- [ ] Blocker 0건
- [ ] Major 0건
- [ ] 모든 수정사항 재검수 완료
- [ ] 최종 Source commit과 EPUB build 기록

상세 운영 기준: #24
